src/account/accountsReducer.js

```diff
@@ -41,6 +43,7 @@ const realmAdd = (state, action) => {
 const realmInit = (state, action) => [
   {
     ...state[0],
+    zulipFeatureLevel: state[0].zulipFeatureLevel,
```

Comment:
Instead of getting the `zulipFeatureLevel` from whatever the state was before this action, we should grab it from the new data that the action includes. In particular, I believe, `action.data.zulip_feature_level`.

Since `action.data` comes from the server data without really being transformed, `zulip_feature_level` might be undefined because the server doesn't provide it. So, we'll want to do, e.g., `action.data.zulip_feature_level ?? 0`.